### ISSUE TITLE
Improve export window

### ISSUE THREAD

**Issue by skllcrn** [wulkano/kap#340]
![app exporting](https://user-images.githubusercontent.com/8440026/35106210-ef4cbe20-fc6d-11e7-8a30-3cfd5fab62c4.png)

Now that #73 in we can move ahead on the design improvements for the export window. That includes displaying how many exports are cued in the title and an option to easily hide the window.

**Comment by sindresorhus** [comment]
When the user clicks "Hide window", what happens? And how do they get the window back?

**Comment by skllcrn** [comment]
We can definitively rely on that, the intention was to make it clear that you can safely hide it even though it's processing.

**Comment by sindresorhus** [comment]
I would say that's implied with it being a normal window.

**Comment by skllcrn** [comment]
![app processing](https://user-images.githubusercontent.com/8440026/35168076-f5d8c582-fd57-11e7-83ba-e14e1ac24aa1.png)

**Comment by sallar** [comment]
@sindresorhus @skllcrn this issue requires implementing multiple progress bars as well right? but this design is different from the one that was being discussed (multiple lines below each other)

**Comment by albinekb** [comment]
No @sallar this is a simplified version of it, as it currently works today it's(or was?) possible to do multiple exports and when you do that there's currently no way to see what's going on 🤔

**Comment by sallar** [comment]
@albinekb Yes what I meant was its not possible to "visually" see whats going on. So which design is the one we should do? This or the multi progress bars?

**Comment by skllcrn** [comment]
Since only one recording is processed at a time we're not doing multiple progress bars for now, opted for a simpler solution where the number of pending exports is displayed in the title.

**Comment by sallar** [comment]
@sindresorhus @skllcrn are moving this progress stuff to a new window? or is that out of scope?

**Comment by skllcrn** [comment]
That sounds like something we could revisit in the upcoming rewrite, while only doing the front-end updates for now.

**Comment by sindresorhus** [comment]
But the popover is currently broken (It shows the main content momentary before starting to export), so would be easier to just move it out into a new window now, than to waste time fixing that.
